iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ShuntCompensatorModification.java

```diff
@@ -59,6 +56,19 @@ public void apply(Network network, NamingStrategy namingStrategy, boolean throwE
         }
     }
 
+    @Override
+    public NetworkModificationImpact hasImpactOnNetwork(Network network) {
+        impact = DEFAULT_IMPACT;
+        ShuntCompensator shuntCompensator = network.getShuntCompensator(shuntCompensatorId);
+        if (shuntCompensator == null) {
+            impact = NetworkModificationImpact.CANNOT_BE_APPLIED;
```

Comment:
do we also want to verify that sectionCount is in the allows [minSectionCount, maxSectionCount] range of the shunt compensator ?